package.json

```diff
@@ -4,8 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
```

Comment:
WHY Did you change NODE_ENV to test

Reply:
NODE_ENV should be set to test when testing.